packages/ember/src/components/form/radio-group.gts

```diff
@@ -0,0 +1,112 @@
+import { hash } from '@ember/helper';
+import { on } from '@ember/modifier';
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+
+import BoundValue from './bound-value.ts';
+
+import type { Optional } from '../../types';
+import type { ComponentLike } from '@glint/template';
+
+export interface RadioGroupFieldSignature {
+  Element: HTMLDivElement;
+  Args: {
+    name: string;
+    basic?: boolean;
+    disabled?: boolean;
+  };
+  Blocks: {
+    default: [{ Radio: ComponentLike<RadioField> }];
+  };
+}
+
+export default class RadioGroupField extends BoundValue<
+  RadioGroupFieldSignature,
+  string
+> {
+  @action
+  change(updatedValue: Optional<string>) {
+    if (updatedValue) {
+      this.value = updatedValue;
+      this.onChange?.(updatedValue);
+    }
+  }
+
+  get classList() {
+    const classes = ['form-control'];
+
+    if (this.args.basic) {
+      classes[0] += '-plaintext';
+    }
+
+    return classes.join(' ');
+  }
+
+  <template>
+    <div class={{this.classList}} ...attributes>
+      {{yield
+        (hash
+          Radio=(component
+            RadioField
+            name=@name
+            disabled=@disabled
+            onChange=this.change
+            currentValue=this.value
+          )
+        )
+      }}
+    </div>
+  </template>
+}
+
+export interface RadioFieldSignature {
+  Element: HTMLInputElement;
+  Args: {
+    name: string;
+    option?: string;
+    currentValue?: string | null;
+    label?: string;
+    disabled?: boolean;
+    onChange?: (value: string, ...args: unknown[]) => void;
```

Comment:
Can you alphabetize these options? You can leave the action at the bottom, that's our current standard.
```suggestion
    currentValue?: string | null;
    disabled?: boolean;
    label?: string;
    name: string;
    option?: string;
    onChange?: (value: string, ...args: unknown[]) => void;
```